libsolidity/formal/SMTEncoder.h

```diff
@@ -403,12 +403,17 @@ class SMTEncoder: public ASTConstVisitor
 
 	/// Creates symbolic expressions for the returned values
 	/// and set them as the components of the symbolic tuple.
-	void createReturnedExpressions(FunctionCall const& _funCall, ContractDefinition const* _contextContract);
+	void createReturnedExpressions(FunctionDefinition const* _funDef, Expression const& _calledExpr);
 
 	/// @returns the symbolic arguments for a function call,
 	/// taking into account attached functions and
 	/// type conversion.
-	std::vector<smtutil::Expression> symbolicArguments(FunctionCall const& _funCall, ContractDefinition const* _contextContract);
+	std::vector<smtutil::Expression> symbolicArguments(
+		FunctionDefinition const* _funDef,
+		Expression const* _calledExpr,
+		FunctionType const* _funType,
+		std::vector<ASTPointer<Expression const>> _arguments
```

Comment:
Is there a reason why this cannot be a `const &`?
```suggestion
		std::vector<ASTPointer<Expression const>> const& _arguments
```

Reply:
Good catch. I also corrected similar problems in BMC and CHC.